src/bci_build/package/nginx/README.md.j2

```diff
@@ -0,0 +1,90 @@
+# {{ image.pretty_name }} Container Image
+
+{% include 'badges.j2' %}
+
+## Description
+
+nginx (pronounced "engine-x") is an open-source reverse proxy server for the HTTP, HTTPS, SMTP, POP3, and IMAP protocols. nginx can also act as a load balancer, HTTP cache, and a web server (origin server).
+
+## Usage
+
+By default, the image launches nginx with the same configuration that comes with the SUSE Linux Enterprise Server.
+
+```ShellSession
+$ podman run -it --rm -p 8080:80 {{ image.pretty_reference }}
+```
+
+Or:
+
+```ShellSession
+$ podman run -it --rm -p 8080:80 -v /path/to/html/:/srv/www/htdocs/:Z {{ image.pretty_reference }}
+```
+
+**Note:** The directory `/srv/www/htdocs/` is the root directory used by the default server. Additional servers can use any other path.
+
+You can access the served content on http://localhost:8080 or http://host-ip:8080.
+
+## Using templates
+
+By default, nginx doesn't support environment variables inside configuration blocks. This image includes a script that can extract environment variables before nginx creates configuration files.
+
+The script reads `.template` files stored in `/etc/nginx/templates/` and saves the result of the [`envsubst`](https://www.gnu.org/software/gettext/manual/html_node/envsubst-Invocation.html) command to the directory `/etc/nginx/conf.d/`.
+
+For example, if you want nginx to use port 80, create a file named `/etc/nginx/templates/default.conf.template` containing the following variable definition:
+
+```nginx
+listen  ${NGINX_PORT};
+```
+
+This makes the script to render the template above to `/etc/nginx/conf.d/default.conf` as follows:
```

Comment:
I don't understand this sentence, could you reword it please?

Reply:
> The template above is then rendered to `/etc/nginx/conf.d/default.conf` as follows:

Is this version understandable?

Reply:
Fixed.